### ISSUE TITLE
训练阶段loss为0，推理阶段输出为空

### ISSUE THREAD

**Issue by Melkor007** [zjukg/KoPA#14]
你好我在复现代码的时候发现训练阶段loss从第二次输出开始全部为0：
{'loss': 1.5336, 'learning_rate': 2.9999999999999997e-05, 'epoch': 0.01}
{'loss': 0.0, 'learning_rate': 5.9999999999999995e-05, 'epoch': 0.01}
{'loss': 0.0, 'learning_rate': 8.999999999999999e-05, 'epoch': 0.02}
同时训练结束在推理阶段运行输出为空，打印具体输出发现内容是类似`<s><unk><unk><unk><unk>`这样的无意义序列。
请问这可能是什么原因导致的？

**Comment by Zhang-Each** [comment]
这种情况一般是模型中出现了NAN，导致模型的参数失效了

**Comment by Melkor007** [comment]
发现问题了，是pytorch和peft安装的版本不对，谢谢
